tests/support/util.tcl

```diff
@@ -555,12 +555,24 @@ proc get_child_pid {idx} {
     return $child_pid
 }
 
+proc statsrstat {key r} {
+    if {[regexp "\r\n$key:(.*?)\r\n" [$r info stats] _ value]} {
+        set _ $value
+    }
+}
+
```

Comment:
there's a method called `s` which does this:
```
[s total_error_replies]
```

Reply:
thank you @oranagra, addressed in the last commit